controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -459,11 +459,9 @@ func (r *OCSInitializationReconciler) ensureOcsOperatorConfigExists(initialData
 			return sc.Spec.AllowRemoteStorageConsumers
 		})
 		if allowConsumers {
-			ocsOperatorConfigData[util.CsiEnableCephFSKey] = "false"
-			ocsOperatorConfigData[util.CsiEnableRBDKey] = "false"
+			ocsOperatorConfigData[util.DisableCSIDriverKey] = "true"
 		} else {
-			ocsOperatorConfigData[util.CsiEnableCephFSKey] = "true"
-			ocsOperatorConfigData[util.CsiEnableRBDKey] = "true"
+			ocsOperatorConfigData[util.DisableCSIDriverKey] = "false"
```

Comment:
we dont need this change, lets keep it as it was before

Reply:
ack, no need for this PR then as of now? or no need to remove already added keys?

Reply:
we just need to set `ocsOperatorConfigData[util.DisableCSIDriverKey] = "true"` this in case of provider mode

Reply:
yes, I was the one who introduced those two keys and so wanted to remove them.